register-service/src/main/resources/sql/20211118_07_alter_slot_table_add_fitness_class.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE slots ADD fitness_class_id BIGINT;
```

Comment:
please add constraints to  fitness_class_id BIGINT; not null and reference to slot table

Reply:
will be added later in scope of https://github.com/javaGuruBY/jis8-project/issues/44